tests/EventListener/ExceptionListenerTest.php

```diff
@@ -64,6 +65,8 @@ public function testHandleException(): void
         self::assertEquals([], $responseBody['context']);
         $record = $handler->getRecords()[0];
         self::assertEquals(Logger::CRITICAL, $record['level']);
+        /** @var array $record['context'] */
```

Comment:
Maybe use an actual assert instead of a phpdoc?

```suggestion
        self::assertIsArray($record['context']);
```

Reply:
sure, dik